src/bidiMapper/modules/cdp/CdpTarget.ts

```diff
@@ -520,6 +540,14 @@ export class CdpTarget {
       .flatMap((script) => script.channels);
   }
 
+  async #updateWindowId() {
+    const {windowId} = await this.#cdpClient.sendCommand(
```

Comment:
Should this be the browser client? 
I am not sure how Browser domain is implemented.